app/models/movie.rb

```diff
@@ -1,3 +1,9 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_and_belongs_to_many :actors
+  
+  def add_actor(actor_id)
+    actor = Actor.find(actor_id)
+    self.actors << actor unless self.actors.include?(actor)
```

Comment:
This is okay to put in a model method but just finding an actor and adding it to a movie is simple CRUD logic that could live in a controller (as opposed to the more complex querying calculations with average, order, etc which we want to live in a model method)